deploy.sh

```diff
@@ -55,6 +55,9 @@ $THIS_DIR/scripts/update_project_version.py public_pom.xml $project_version > ge
 
 mvn deploy ${MVN_OPTIONS[@]} -Dossrh-deploy
 
+# Run WhiteSource script
```

Comment:
You might want to `chmod 755 whitesource/run_whitesource.sh`

Reply:
Thanks, I've updated the file using `update-index`, it should be marked as executable now.